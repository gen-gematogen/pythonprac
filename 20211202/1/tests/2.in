class C(metaclass=dump):
	def __init__(self, rep = 1):
		self.rep = rep
		print("meta" * rep)
	def alien_mark(self, val):
		if (self.rep^val)%2:
			return "sure"
		return "maybe"

c = C(3)
print(c.alien_mark(1001))
