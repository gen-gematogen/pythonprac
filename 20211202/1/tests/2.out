__init__: (3,), {}
metametameta
alien_mark: (1001,), {}
maybe
